Janine.chan/docs 11356 cloudcraft sensitive data (#30191)
DataDog/documentation: content/en/datadog_cloudcraft/overlays.md

- Analyze findings in context, so you can prioritize remediation:
  - Misconfigurations
  - Identity risks
  - Sensitive data (S3 buckets only)
- Assess your security posture before deploying applications

This view is ideal for mapping attack surfaces during penetration tests or security audits. To keep the diagram focused, components like EBS volumes and NAT Gateways are excluded. 

#### Investigate misconfigurations and identity risks

By default, the security overlay shows Critical, High, and Medium severity misconfigurations or identity risks, which you can filter in the legend.

You can click on any resource that has findings to open a side panel with more details, allowing deeper investigation without leaving the diagram. Click **Investigate** to get more context about the finding and learn how to remediate it.

{{< img src="datadog_cloudcraft/overlays/cloudcraft_security_overlay_with_ccm_3.png" alt="Security overlay in Cloudcraft with the misconfigurations filter applied, highlighting the collapsible legend in the bottom left of the screen, and the Investigate button on the side panel." style="width:100%;" >}}

#### Investigate sensitive data

> You can view sensitive data matches for your S3 buckets. Click a resource that has matches to learn more about the sensitive data matches for that bucket. Then, hover over a filename and click the **Inspect in AWS** button that appears to address it.

Reviewer: ```suggestion
You can view sensitive data matches for your S3 buckets. Click a resource with matches to learn more about the bucket's sensitive data matches. Then, hover over a filename and click its **Inspect in AWS** button.
```

Reviewer: "that appears to address it" seemed like it might or might not address it. I think you mean the button that appears, so I'm just suggesting a simple change.
